src/classic/element/ReactElement.js

```diff
@@ -103,7 +103,7 @@ var ReactElement = function(type, key, ref, owner, context, props) {
   // through the owner.
   this._context = context;
 
-  if (__DEV__) {
+  if (__DEBUG__) {
```

Comment:
the freezing done in this branch may have nontrivial perf cost.  what's the value of `useMutationMembrane` here?

We might want to keep this off in our debug build.